flake.nix

```diff
@@ -225,21 +227,24 @@
             buildInputs = [ pkgs.python3 ];
           };
 
-          buildPackage = { pname, extraArgs ? { } }:
+          buildPackage = { pname, extraBuildArgs ? "", extraArgs ? { } }:
             craneLib.buildPackage ({
               inherit
                 pname
                 src
                 version
                 cargoArtifacts;
 
-              cargoExtraArgs = "${cargoBuildExtraArgs} --package ${pname}";
+              cargoExtraArgs = "${cargoBuildExtraArgs} ${extraBuildArgs} --package ${pname}";
             } // extraArgs);
         in
         rec {
           inherit cargoArtifacts;
           nickel-lang-core = buildPackage { pname = "nickel-lang-core"; };
-          nickel-lang-cli = buildPackage { pname = "nickel-lang-cli"; };
+          nickel-lang-cli = buildPackage {
+            pname = "nickel-lang-cli";
+            extraBuildArgs = "--features format";
```

Comment:
Don't we want to disable it by default for this PR, until we figure out the static binary build story?

Reply:
We can keep it enabled for the normal CLI build, and just disable it for the static binary. That way at least `nix run github:tweag/nickel` gets the formatting feature.

Reply:
Ah, what I understood as "principle of least surprise" from the last planning meeting was that the binary built/run from nix with the default command and the one provided as a static binary shouldn't differ, as you should expect as a user. I would vote for disabling everywhere for now, but just merge it, so that this PR doesn't drift away from the master branch.

Reply:
It's what I meant. But I don't think you need to choose how you're going to handle it before you release.

Reply:
Alright, I've disabled the `format` feature for the nix build 👍